src/info/manifest.c

```diff
@@ -1552,6 +2322,55 @@ manifestFileTotal(const Manifest *this)
     FUNCTION_TEST_RETURN(lstSize(this->fileList));
 }
 
+void
+manifestFileUpdate(
+    Manifest *this, const String *name, uint64_t size, uint64_t sizeRepo, const char *checksumSha1, const String *reference,
+    bool checksumPage, bool checksumPageError, const VariantList *checksumPageErrorList)
+{
+    FUNCTION_TEST_BEGIN();
+        FUNCTION_TEST_PARAM(MANIFEST, this);
+        FUNCTION_TEST_PARAM(STRING, name);
+        FUNCTION_TEST_PARAM(UINT64, size);
+        FUNCTION_TEST_PARAM(UINT64, sizeRepo);
+        FUNCTION_TEST_PARAM(STRINGZ, checksumSha1);
+        FUNCTION_TEST_PARAM(STRING, reference);
+        FUNCTION_TEST_PARAM(BOOL, checksumPage);
+        FUNCTION_TEST_PARAM(BOOL, checksumPageError);
+        FUNCTION_TEST_PARAM(VARIANT_LIST, checksumPageErrorList);
+    FUNCTION_TEST_END();
+
+    ASSERT(this != NULL);
+    ASSERT(name != NULL);
+    ASSERT(
+        (!checksumPage && !checksumPageError && checksumPageErrorList == NULL) ||
+        (checksumPage && !checksumPageError && checksumPageErrorList == NULL) || (checksumPage && checksumPageError));
+
+    ManifestFile *file = (ManifestFile *)manifestFileFind(this, name);
+
```

Comment:
Do we no longer care about copying the "master" setting from the prior manifest for the file?

Reply:
We do not.  We just recalculate master for each manifest.  It's doubtful that we should even waste space storing it, but that's the format.  This was only *really* being saved for expect log testing purposes.